wallet/src/de/schildbach/wallet/ui/backup/BackupWalletDialogFragment.java

```diff
@@ -253,17 +247,12 @@ public void onDismiss(final DialogInterface dialog) {
         passwordAgainView.removeTextChangedListener(textWatcher);
 
         showView.setOnCheckedChangeListener(null);
+        alertDialog.dismiss();
 
         wipePasswords();
         super.onDismiss(dialog);
     }
 
-    @Override
-    public void onCancel(final DialogInterface dialog) {
-        activity.finish();
```

Comment:
Not sure why we were finishing `SecurityActivity` on the dialog dismissal. Seems like just dismissing is fine.

Reply:
This was taken from Bitcoin Wallet, where I think an activity was created that displays the backup wallet dialog.  

However, we are not creating an activity to only display this dialog:

Reference:
https://github.com/bitcoin-wallet/bitcoin-wallet/blob/main/wallet/src/de/schildbach/wallet/ui/backup/BackupWalletActivity.java#L28-L40